addons/sale/models/sale_order.py

```diff
@@ -258,6 +258,7 @@ def _search_invoice_ids(self, operator, value):
         compute_sudo=True,
         help="Technical field to filter the available taxes depending on the fiscal country and fiscal position.")
     company_id = fields.Many2one('res.company', 'Company', required=True, index=True, default=lambda self: self.env.company)
+    country_code = fields.Char(related='company_id.account_fiscal_country_id.code', string="Country code")
```

Comment:
`company_fiscal_country_code` instead ?
I find the name `country_code` ambiguous, it could be the partner country code, the company country code, ...

Reply:
It is the name we have been using on all models since version 13.0 (before the concept of fiscal countries was added), I would like to keep the same name everywhere

Reply:
The argument 'we always did it like that' doesn't mean it was a good idea and that we have to keep doing the same ...

Adding 'useless' fields in core modules (only used when a specific obscure module is installed) is already a bad practice IMHO, but if it's to have confusing field names & content, then please keep it in `l10n_in_sale`.  At worst, the field is defined in multiple modules (If one day there is another `l10n_xx_sale`), but AFAIK, the ORM should still correctly cover the case (and even then, you could move it in a `l10n_sale` module).

Reply:
> The argument 'we always did it like that' doesn't mean it was a good idea and that we have to keep doing the same ...

I know, maybe we can change it to all have another name. But it should all be the same name.